README.md

```diff
@@ -33,6 +33,12 @@ Please submit a pull request if there is any material that you think should be i
     * [Hash Table](#hash-table)
     * [Bloom Filter](#bloom-filter)
 * [DDL](#ddl)
+* [Relational Model](#relational-model)
+    * [Codd's Rules](codd's-rules)
+    * [Relational Data Model](relational-data-model)
+    * [Relational Algebra](relational-algebra)
+    * [ER to Relational Model](er-to-realtional-model)
+    * [SQL - Overview](sql---overview)
```

Comment:
@Shree-Gillorkar thanks for your contribution, could you add some paper or books about these topics?